O/Openresty/build_tarballs.jl

```diff
@@ -0,0 +1,54 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "Openresty"
+version = v"1.15.8"
+
+# Collection of sources required to complete build
+sources = [
+    ArchiveSource("https://openresty.org/download/openresty-1.15.8.3.tar.gz", "b68cf3aa7878db16771c96d9af9887ce11f3e96a1e5e68755637ecaff75134a8"),
+    ArchiveSource("https://ftp.pcre.org/pub/pcre/pcre-8.43.tar.gz", "0b8e7465dc5e98c757cc3650a20a7843ee4c3edf50aaf60bb33fd879690d2c73"),
+    ArchiveSource("https://www.openssl.org/source/openssl-1.0.2t.tar.gz", "14cb464efe7ac6b54799b34456bd69558a749a4931ecfd9cf9f71d7881cac7bc"),
+    ArchiveSource("https://www.zlib.net/zlib-1.2.11.tar.gz", "c3e5e9fdd5004dcb542feda5ee4f0ff0744628baf8ed2dd5d66f8ca1197cb1a1")
```

Comment:
Why are these used as sources?  We have all of them as JLL packages already

Reply:
Openresty/Nginx needs these sources for compilation (ref: https://docs.nginx.com/nginx/admin-guide/installing-nginx/installing-nginx-open-source/#sources). The makefile shows that it links statically to these libraries and has targets that expect source folders and build each of them.